frontend/components/TableContainer/DataTable/DataTable.tsx

```diff
@@ -495,7 +499,13 @@ const DataTable = ({
             {headerGroups.map((headerGroup) => (
               <tr {...headerGroup.getHeaderGroupProps()}>
                 {headerGroup.headers.map((column) => {
-                  let thProps = column.getSortByToggleProps();
+                  // column.title exists but is rendering a typescript error that it doesn't
```

Comment:
@lukeheath here's the issue.

Line 507 removes all "Toggle SortBy" tooltips, if we could get line 506 that's commented out to work, it would allow for removing tooltips on columns that do not have a defined title.